### ISSUE TITLE
Unable to get static query data on Windows

### ISSUE THREAD

**Issue by mohsenkhanpour** [ehrencrona/gatsby-plugin-testing#4]
I have set up testing for a project, and I work on it both on Linux and Windows,
all the tests that have static-query pass without problem on Linux,
however on Windows static queries are not loaded and it throws. 
```
● Index › renders with fixtures

    While getting static query data: Did not find component C:\Users\xx\xx\project\src\components\Sections\Index\IELTS\Courses.tsx, only: 
	C:/Users/xx/xx/project/src/components/Sections/Index/IELTS/Courses.tsx
    C:/Users/xx/xx/project/src/components/Sections/Index/Speaking/Courses.tsx

 Do you need to re-run gatsby build?
          at getQueryHashForComponentPath (C:\Users\xx\xx\project\node_modules\gatsby-plugin-testing\src\static-query.js:75:11)
```

I have run `gatsby build`, the page queries have no problem, the same test runs flawlessly on Linux.

It seems like it finds the component, but there's a problem with handling of slashes/backslashes.

**Comment by ehrencrona** [comment]
Unfortunately, I don't have any Windows machine. Based on what I see in the error message, I've tried to patch things up so they might work on Windows. Could you please try using [the branch `windows`](https://github.com/ehrencrona/gatsby-plugin-testing/tree/windows) and see if it fixes the issue?

**Comment by mohsenkhanpour** [comment]
Thanks,
I will definitely check this branch and inform you how it goes.

**Comment by mohsenkhanpour** [comment]
@ehrencrona The branch didn't fix the issue, however I saw what you had done, and could make it work with a slight change.

```js
function getQueryHashForComponentPath(componentPath) {
 	const components = Object.values(readStaticQueries());

	// on windows, componentPath has backward slashes but the query hash forward slashes
	const normalizedPath = sep == "\\" ? componentPath.replace(/\\/g, "/") : componentPath
  	const component = components.find((c) => c.componentPath == normalizedPath);

  if (!component) {
    throw new Error(
      `While getting static query data: Did not find component ${componentPath}, only: ${components
		.map((c) => c.componentPath)
        .join("\n")}\nDo you need to re-run gatsby build?`
    );
  }

  return component.hash;
}
```

You were checking for file paths after the error was fired. I moved the logic outside the error closure and applied it to `componentPath` and it fixed the problem.

**Comment by mohsenkhanpour** [comment]
I submitted a PR with suggested changes.

**Comment by ehrencrona** [comment]
Awesome, thanks! I released this as 0.3.3
